lib/usage.js

```diff
@@ -128,12 +128,14 @@ module.exports = function (yargs, y18n) {
   self.help = function () {
     normalizeAliases()
 
-    var demanded = yargs.getDemanded()
+    var demandedOption = yargs.getDemandedOption()
+    var demandedCommand = yargs.getDemandedCommand()
```

Comment:
Maybe we should rename those to make them plurals? Like `demandedOptions`, `getDemandedOptions` etc.?